webrecorder/browsertrix
# Get registration enabled settings from endpoint

Retrieve frontend app setting on registration from backend endpoint.

### Manual testing
1. Run app with `yarn start`. Verify that you can access http://localhost:9870/sign-up
2. Add `body.registrationEnabled = false;` to line 140 of index.ts

### TODO
- Replace hardcoded token expiration with expiration field from JWT (https://github.com/ikreymer/browsertrix-cloud/issues/66)
- Add unit test case (https://github.com/ikreymer/browsertrix-cloud/projects/1#card-74313814)
